.changeset/cuddly-chefs-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/application-config': patch
+---
+
+Define new field `additionalOAuthScopes` in the Custom Application config.
```

Comment:
Should we describe what this new field should be use for?
Or maybe add a link to the updated documentation?